src/QMCWaveFunctions/Fermion/MultiDiracDeterminant.h

```diff
@@ -83,7 +83,7 @@ class MultiDiracDeterminant : public WaveFunctionComponent
    *
    * Note: ciConfigList should have been populated when calling this function
    */
-  void set(int first, int nel, int ref_det_id);
+  void set(int first, int nel, int ref_det_id, std::vector<size_t>& C2nodes_ptcl);
```

Comment:
Update documentation.

Reply:
added description of `C2nodes_ptcl` to documentation